src/test/regress/sql/multi_name_nested.sql

```diff
@@ -0,0 +1,20 @@
+create table namenest1 (id integer primary key, user_id integer);
+create table namenest2 (id integer primary key, value_2 integer);
+select * from create_distributed_table('namenest1', 'id');
+select * from create_reference_table('namenest2');
+
+SELECT r
```

Comment:
Please add a comment explaining what this specific query is testing. It is very difficult to figure that out without having much context about the issue and by just looking at the query. People shouldn't need to spend more than a minute or two to figure out what each query is testing. See other tests for examples of comments, and the style we use.